### PR TITLE
HEEDLS-445 EditCourseOptions

### PR DESCRIPTION
**Pull request by showkath10**
### JIRA link
https://softwiretech.atlassian.net/browse/HEEDLS-445

### Description
Added a new page for editing course options
TrackingSystem-> Centre course setup -> Manage Course -> Edit

### Screenshots
image
Screenshot 2021-09-24 040640

-----
### Developer checks
I have:
- [x ] Run the formatter and made sure there are no IDE errors.
- [x ] Written tests for the changes (controller, data services, services, view models etc) and manually tested my work with and without JavaScript. Full manual testing guidelines can be found here: https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/6703648740/Testing
- [ ] Updated/added documentation in Swiki and/or Readme.
- [x ] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken.
- [x ] Scanned over my own MR to ensure everything is as expected.

### CODE REVIEW

File: DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs [hunk 1]
```diff
@@ -49,5 +49,51 @@ public void Index_returns_ManageCourse_page_when_appropriate_course_found()
...
+
+            // When
+            var result = controller.EditCourseOptions(1);
+
+            // Then
+            result.Should().BeViewResult().WithDefaultViewName().ModelAs<EditCourseOptionsViewModel>();
+        }
+
+        [Test]
+        public void Edit_Course_Options_page_when_coursed_details_are_not_updated()
```
Comment (JBsoftwire) [inline DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs:68]: Typo 'coursed' here and on the test below.

**Review by JBsoftwire — CHANGES_REQUESTED** [review]
Found a typo, but no changes from me besides that. Could we get a mobile view screenshot though?

File: DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs (hunk 1 above)
Comment (showkath10) [reply DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs:68]: Sorry for typo. this has been corrected.

Comment (JBsoftwire) [reply DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CourseSetup/ManageCourseControllerTests.cs:68]: I think you still need to push the commit with the fix.

**Comment by showkath10** [comment]
there are some merge errors

**Pull request closed without merging**